activities.json

```diff
@@ -1069,5 +1069,16 @@
     "org": "WHATWG",
     "title": "timezonechange event",
     "url": "https://github.com/whatwg/html/pull/3047"
+  },
+  {
+    "ciuName": "webgpu",
+    "description": "This specification describes support for accessing modern 3D graphics and GPU computation capabilities on the Web.",
+    "mozBugUrl": "https://bugzilla.mozilla.org/show_bug.cgi?id=1602129",
+    "mozPosition": "important",
+    "mozPositionDetail": "We believe that WebGPU is key to enable more interactive and content-rich applications on the Web than currently possible with WebGL. It's an API that is future-looking and can scale better to the capabilities of GPUs, with less overhead for users. Firefox Nightly has a prototype implementation behind a pref that was able to run the WebGPU examples as of April 2020.",
```

Comment:
A few comments here:

First, we've mostly been trying to keep standards-positions *separate* from Firefox product planning.  So to be consistent with that -- and the way we've phrased other positions, I'd probably drop the last sentence here.

Second, I worry that the remaining sentences sound a little more like marketing for the spec than the voice we usually use in positions.  I'd perhaps drop the "It's an API that is future looking and" part, and maybe expand a little bit (but not too much -- maybe a sentence) on why it better fits the capabilities of GPUs and has less overhead?

Reply:
That sounds reasonable. I partially rewrote the description, please take another look!